src/WebJobs.Extensions.DurableTask/DurableOrchestrationClient.cs

```diff
@@ -48,6 +48,21 @@ public HttpResponseMessage CreateCheckStatusResponse(HttpRequestMessage request,
             return this.config.CreateCheckStatusResponse(request, instanceId, this.attribute);
         }
 
+
+        /// <summary>
+        /// Creates an HTTP response for checking the status of the specified instance supporting synchronous response as well. 
+        /// </summary>
+        /// <param name="request">The HTTP request that triggered the current function.</param>
+        /// <param name="instanceId">The unique ID of the instance to check.</param>
+        /// <param name="totalTimeout">Total allowed timeout for output from the durable function</param>
```

Comment:
For consistency, each parameter description and summary description should end in a period.